tests/worktree/worktree.c

```diff
@@ -228,6 +228,30 @@ void test_worktree_worktree__init(void)
 	git_repository_free(repo);
 }
 
+void test_worktree_worktree__add_from_bare(void)
+{
+	git_worktree *wt;
+	git_repository *repo, *wtrepo;
+	git_buf path = GIT_BUF_INIT;
+
+	repo = cl_git_sandbox_init("testrepo.git");
+
+	cl_assert_equal_i(1, git_repository_is_bare(repo));
+	cl_assert_equal_i(0, git_repository_is_worktree(repo));
+
+	cl_git_pass(git_buf_joinpath(&path, repo->gitdir, "../worktree-frombare"));
```

Comment:
You should be able to just use "./worktree-frombare" instead of assembling the path, as we always chdir into the test data directory.